app/views/reports/elements/_my_module_repository_element.html.erb

```diff
@@ -13,6 +13,14 @@
       <div class="pull-left repository-name">
         <%=t "projects.reports.elements.module_repository.name", repository: repository.name, my_module: my_module.name %>
       </div>
+      <% if defined? export_all and export_all%>
```

Comment:
Missing space on the right `%`. You could use something like this though https://api.rubyonrails.org/v3.2/classes/ActionView/Base.html#label-Passing+local+variables+to+sub+templates.

Reply:
Do you mean the usage of `local_assigns.has_key?`? This would produce even more code. If not, what do you mean exactly?